week-2/14-align-self/flexbox.css

```diff
@@ -0,0 +1,13 @@
+.countries {
+	display: flex;
+}
+/* Add your own CSS code below */
+
+.countries--first {
```

Comment:
Is this (and the code below it) the answer? (I always accidentally commit the answer. :laughing:)

Reply:
.....maybe 😆